misc/landing-page/src/styles/global.css

```diff
@@ -1,41 +1,51 @@
 @import "tailwindcss";
 @import "@xterm/xterm";
 @theme {
-    --color-*: initial;
-    --color-white: #fff;
-    --color-light-20: #fffdf5;
-    --color-light-30: #fffaf7;
-    --color-orange-100: #ef7c00;
-    --color-orange-weird: #ffe6cc;
-    --color-brown-header: #58330c;
-    --color-brown-100: #36220d;
-    --color-brown-90: #423221;
-    --color-brown-70: #5f4122;
-    --color-brown-gray: #696057;
-    --color-black: #301b05;
-    --font-handjet: "Handjet Variable", system-ui;
-    --font-inter: "Inter Variable", sans-serif;
+  --color-*: initial;
+  --color-white: #fff;
+  --color-light-20: #fffdf5;
+  --color-light-30: #fffaf7;
+  --color-orange-100: #ef7c00;
+  --color-orange-weird: #ffe6cc;
+  --color-brown-header: #58330c;
+  --color-brown-100: #36220d;
+  --color-brown-90: #423221;
+  --color-brown-70: #5f4122;
+  --color-brown-gray: #696057;
+  --color-black: #301b05;
+  --font-handjet: "Handjet Variable", system-ui;
+  --font-inter: "Inter Variable", sans-serif;
 
-    --animate-marquee: marquee 12s linear infinite;
-    --animate-marquee-reset: marquee-reset;
+  --animate-marquee: marquee 12s linear infinite;
+  --animate-marquee-reset: marquee-reset;
 
-    @keyframes marquee {
-        0% {
-            transform: translate(0%, 0);
-        }
-        50% {
-            transform: translate(-100%, 0);
-        }
-        50.001% {
-            transform: translate(100%, 0);
-        }
-        100% {
-            transform: translate(0%, 0);
-        }
+  @keyframes marquee {
+    0% {
+      transform: translate(0%, 0);
     }
-    @keyframes marquee-reset {
-        0% {
-            transform: translate(0%, 0);
-        }
+    50% {
+      transform: translate(-100%, 0);
     }
+    50.001% {
+      transform: translate(100%, 0);
+    }
+    100% {
+      transform: translate(0%, 0);
+    }
+  }
+  @keyframes marquee-reset {
+    0% {
+      transform: translate(0%, 0);
+    }
+  }
+}
+
+@layer components {
+  .break {
+    @apply hidden md:block;
+  }
+
+  .header-title {
+    @apply max-w-5xl md:text-[200px] text-[80px] font-handjet font-semibold tracking-normal text-center leading-none text-orange-100;
+  }
```

Comment:
I think we can extract it to component instead, having it here is confusing if you don't know where to look.